src/main/java/com/mxf/rest/entity/ABaseEntity.java

```diff
@@ -0,0 +1,7 @@
+package com.mxf.rest.entity;
+
+public abstract class ABaseEntity {
```

Comment:
What is the reason to create abstract class on the case? to have only one abstract method ? really? Why we can't use interface?